api/test/unit/validation/validationMiddleware.test.js

```diff
@@ -365,4 +365,209 @@ describe('Validate Policy JSON Schema structure', function() {
       done();
     });
   });
+  it('should successfully validate policy schema without start_date and end_date in recurring_schedule', function(done) {
```

Comment:
There is a problem with all these tests.
If I change any of the expects which should cause a failure, it passes.

I was also expecting that I could have 0 recurring or specific dates and pass which doesn't seem to be the case.